src/app/pages/test-report/cat-b/components/manoeuvres/__tests__/manoeuvres.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { By } from '@angular/platform-browser';
+import { AppModule } from '@app/app.module';
+import { DangerousFaultBadgeComponent } from '@components/common/dangerous-fault-badge/dangerous-fault-badge';
+import { DrivingFaultsBadgeComponent } from '@components/common/driving-faults-badge/driving-faults-badge';
+import { SeriousFaultBadgeComponent } from '@components/common/serious-fault-badge/serious-fault-badge';
+import { TickIndicatorComponent } from '@components/common/tick-indicator/tick-indicator';
+import { IonicModule } from '@ionic/angular';
+import { StoreModule } from '@ngrx/store';
+import { configureTestSuite } from 'ng-bullet';
+import { MockComponent } from 'ng-mocks';
+import { DateTimeProvider } from '../../../../../../providers/date-time/date-time';
+import { DateTimeProviderMock } from '../../../../../../providers/date-time/__mocks__/date-time.mock';
+import { ManoeuvresComponent } from '../manoeuvres';
+
+describe('ManoeuvresComponent', () => {
+  let fixture: ComponentFixture<ManoeuvresComponent>;
+  let component: ManoeuvresComponent;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        ManoeuvresComponent,
+        TickIndicatorComponent,
+        MockComponent(DrivingFaultsBadgeComponent),
+        MockComponent(SeriousFaultBadgeComponent),
+        MockComponent(DangerousFaultBadgeComponent),
+      ],
+      imports: [
+        IonicModule,
+        AppModule,
+        StoreModule.forRoot({
+          tests: () => ({
+            currentTest: {
+              slotId: '123',
+            },
+            testStatus: {},
+            startedTests: {
+              123: {
+                testData: {
+                  dangerousFaults: {},
+                  drivingFaults: {},
+                  manoeuvres: {},
+                  seriousFaults: {},
+                  testRequirements: {},
+                  ETA: {},
+                  eco: {},
+                  vehicleChecks: {
+                    showMeQuestion: {
+                      code: 'S3',
+                      description: '',
+                      outcome: '',
+                    },
+                    tellMeQuestion: {
+                      code: '',
+                      description: '',
+                      outcome: '',
+                    },
+                  },
+                  eyesightTest: {},
+                },
+                postTestDeclarations: {
+                  healthDeclarationAccepted: false,
+                  passCertificateNumberReceived: false,
+                  postTestSignature: '',
+                },
+                journalData: {},
+                communicationPreferences: {
+                  updatedEmail: '',
+                  communicationMethod: 'Not provided',
+                  conductedLanguage: 'Not provided',
+                },
+              },
+            },
+          }),
+          // @TODO introduce this once Cat B NGRX introduced
```

Comment:
do we have a story number for this?

Reply:
nice spot, matts ngrx has gone in now, will amend